utils.go

```diff
@@ -21,6 +21,13 @@ func contains(items []string, value string) bool {
 	return false
 }
 
+func fallbackString(value, fallback string) string {
```

Comment:
duplicated func

Reply:
I had trouble making this function reusable (i.e. exported) as they are used in two separate packages.

I'll make another attempt though

Reply:
New package: `internal/or`